File: 1365. How Many Numbers Are Smaller Than the Current Number/1365. How Many Numbers Are Smaller Than the Current Number.py
class Solution:
    def smallerNumbersThanCurrent(self, nums: List[int]) -> List[int]:
        length=len(nums)
        result=[]
        for i in range(length):
            count=0
            for j in range(length):
                if(i!=j and nums[i]>nums[j]):
                    count+=1
            result.append(count)
        return result

File: 1365. How Many Numbers Are Smaller Than the Current Number/1365. How Many Numbers Are Smaller Than the Current Number.java
class Solution {
    public int[] smallerNumbersThanCurrent(int[] nums) {
        int n = nums.length;
        int [] result = new int[n];
        int count;
        for(int i=0; i<n;i++){
            count=0;
            for(int j=0; j<n; j++){
                if(i!=j && nums[i]>nums[j]){
                    count++;
                }
            }
            result[i]=count;
        }
        return result;        
    }
}
